pkg/strace/syscall_linux_amd64.go

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build (linux && arm64) || (linux && amd64)
```

Comment:
Are we really need arm64 part here? We have suffix '_linux_amd64' but in tags we also have arm64. Not sure how it works, can we compile this file for arm64? 

Maybe we can remove architecture suffix, use build tags and so we don't need to copy/paste a file from amd64? I see comment `This is the amd64 syscall map.` in arm64 file.

Reply:
this is an error on my part, I'd prefer to keep the suffix, so will fix. Thanks!

Reply:
(I recall, from previous discussion / review, we prefer to use suffix to indicate go arch and go os).